measurement/ftrace/mem/drtracer.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Trace direct reclaim latency.
+
+import os
+import logging
+
+from measurement import util
+from measurement.files import fileutils
+
+
+class DirectReclaimTracer():
+    # options about tracer
+    ftrace_options = {}
+
+    # output dir
+    data_dir = "mem/drtrace"
+
+    # tracefs mount point
+    tracefs = "/sys/kernel/debug/tracing"
+    direct_reclaim_begin = "events/vmscan/mm_vmscan_direct_reclaim_begin/enable"
+    direct_reclaim_end = "events/vmscan/mm_vmscan_direct_reclaim_end/enable"
+
+    def __init__(self, options={}):
+        self.ftrace_options = options
+
+    def save_trace(self, outputdir=None):
+        _, stderr = util.run_cmd(["cd", self.tracefs])
+        if stderr:
+            logging.fatal("""ERROR: accessing tracing. Root user? Kernel has FTRACE?
+            debugfs mounted? (mount -t debugfs debugfs /sys/kernel/debug)""")
+            return
+
+        orginalwd = os.getcwd()
```

Comment:
We have a `util.cwd()` that may be useful. Or should we just put `originalwd` to a global variable (like in class `Insight`)? Will `chdir()` like this be usual in the future?